style.css

```diff
@@ -26,20 +26,88 @@ body {
   align-items: center;
 }
 
-.logo p {
+.navbar .my-logo {
   color: var(--primary-b-400, #6070ff);
   font-size: 18px;
   font-weight: 700;
   line-height: 20px;
   text-align: center;
   margin-left: 12px;
+  margin-top: 12px;
   padding: 10px 12px;
+  text-decoration: none;
 }
 
-.menu-icon img {
-  margin-right: 24px;
-  padding: 8px;
-  vertical-align: middle;
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 30px;
+}
+
```

Comment:
- [OPTIONAL] This wasn't mentioned by the previous reviewer, but as you are building out your portfolio, it should still be responsive and look great across multiple screens.
- - Looking at the video below, you can see that your menu is not fully responsive. At some point, some part of the menu tends to enter into the screen, and also at some point, the styling on the menu isn't applied. I'll suggest you rectify this as your menu and portfolio should look great as you build it out. :smile: :wave: :+1: 

- - Video reference:

https://github.com/78fatemanazari/Portfolio/assets/107632780/273e112c-b924-4653-ad36-9d0cefaccbc5

- [OPTIONAL] You can consider adding a smooth scroll to your project, it will make navigating to different sections when you click on a menu item smooth. You can achieve this by adding this at the top of your CSS file. :+1:  `html {
  scroll-behavior: smooth;
}`